lit/docs/operation/security-hardening.lit

```diff
@@ -0,0 +1,65 @@
+\title{Security Hardening}{security-hardening}
+
+\use-plugin{concourse-docs}
+
+Concourse tasks run in containers, which provide a degree of isolation from the
+host. However, if inadequate attention is paid to security, anyone with the
+ability to update pipelines or modify a script called in a task might be
+able to escape from the container and take control of the host. From there,
+they could access other host resources, interfere with pipelines they might
+not otherwise have access to, and collect credentials.
+
+Following the guidance in this section can help you to greatly reduce the risk
+of a container escape.
+
+\section{
+  \title{Keeping your kernel up-to-date}
+
+  Containers run in different Linux namespaces on the same Linux kernel as the
+  host system. Vulnerabilities in the kernel version you run can easily allow
```

Comment:
I'd reword this, it's not exactly 'easy' nor is it common.

Reply:
I have pushed another commit to drop the word easily.